.github/workflows/package-ci.yml

```diff
@@ -369,3 +369,56 @@ jobs:
           path: |
             oxidation/client/electron/dist/client_*_*.snap
             oxidation/client/electron/dist/client-*.AppImage
+
+  package-android-app:
+    runs-on: ubuntu-22.04
+    name: 🤖 Package Android app
+    env:
+      NDK_VERSION: 22.1.7171670
+      SDK_VERSION: 30.0.3
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # pin v3.0.2
+
+      - name: Install rust toolchain
+        id: rust-toolchain
+        uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f # pin v1.0.6
+        with:
+          toolchain: ${{ env.rust-version }}
+          profile: minimal
+          override: true
+
+      - name: Install additional targets for Rust
+        run: |
+          set -x
+          rustup target add armv7-linux-androideabi
+          rustup target add i686-linux-android
+          rustup target add aarch64-linux-android
+          rustup target add x86_64-linux-android
+
+      - name: Configure Runner environment
+        run: |
+          echo "$ANDROID_HOME/cmdline-tools/latest/bin" > $GITHUB_PATH
+
+      - name: Install dependencies for Android
+        run: sdkmanager --install "ndk;${{ env.NDK_VERSION }}" "build-tools;${{ env.SDK_VERSION }}"
+
+      - name: Install dependencies for ionic project
+        run: npm clean-install
+        working-directory: oxidation/client
+
+      - name: Build ionic for Android
+        run: |
+          npx ionic build android --no-open --prod
```

Comment:
It would be better to include this into `client/package.json` so that we could do `npm run android:prod-copy` / `npm run electron:prod-copy` etc.